### ISSUE TITLE
bower.json don't declare the main file in the "main" section

### ISSUE THREAD

**Issue by darlanalves** [Foxandxss/angular-toastr#59]
bower.json don't have a reference to dist/angular-toastr.js, so automated tools like the wiredep module won't add the main file to app's dependency list.

**Comment by Foxandxss** [comment]
It is declaring the tpls version which is the one needed. The other is for the ones who dont need the built-in template. I think I cannot have both in there

**Comment by darlanalves** [comment]
Le't me check again. I added it in the local copy and it worked. Without that declaration, it fails.

**Comment by Foxandxss** [comment]
Split them how?

**Comment by darlanalves** [comment]
Sorry, forgot to answer that day.

So, I installed it via bower and gone to bower.json to see what files should I include, then I saw only the `angular-toastr.tpls.js` file, which make me think that only the templates were included on bower, not the lib itself. Then, I opened the file and realized it was both the lib and the templates. For the sake of precision, split it in two files, `angular-toastr.js` and `angular-toastr.tpls.js`.

Otherwise, nevermind. It's working like a charm :)
